### PR TITLE
Add ip to geometry node

### PR DESCRIPTION
**Pull request by wybert**
Added the ip to the geometry node, may need server thing to be done

- [x] input schema
- [x] add more options
- [x] Keep column name unique
- [x] Limit the column name if needed. Now I append the city, region, country, long, lat, and geometry columns
- [x] add an icon for it

### CODE REVIEW

**Comment by wybert** [comment]
This could solve this issue #41

**Comment by wybert** [comment]
Notice a Python package called [ip2geotools](https://github.com/tomas-net/ip2geotools) for various services for free and commercial providers. But it seems that it haven't been update recently.

**Comment by koettert** [comment]
@UrbanGISer can you please create a beautiful icon for this new node

File: knime_extension/src/nodes/conversion.py [hunk 1]
```diff
@@ -910,6 +910,304 @@ def execute(self, exec_context: knext.ExecutionContext, input_table):
         return knut.to_table(gdf)
 
 
+############################################
+# ip to geometry
+############################################
+@knext.node(
+    name="IP to Geometry",
+    node_type=knext.NodeType.MANIPULATOR,
+    icon_path=__NODE_ICON_PATH + "IPToGeo.png",
```
Comment (koettert) [inline knime_extension/src/nodes/conversion.py:919]: @wybert can you please also include the node icon into the pull request

**Review by koettert — CHANGES_REQUESTED** [review]
Please review my changes and add the missing icon. Once done you can merge to master.

**Comment by koettert** [comment]
Please add the missing icon and merge the pull request if you are happy with my changes @wybert

**Comment by wybert** [comment]
> 

@UrbanGISer I need your help to add an AWESOME icon

**Comment by wybert** [comment]
@koettert Lingbo added an icon for this, you could merge the pr now:)

**Pull request merged**